pkg/cli/alpha/internal/update/update.go

```diff
@@ -334,8 +336,12 @@ func (opts *Update) mergeOriginalToUpgrade() error {
 		var exitErr *exec.ExitError
 		// If the merge has an error that is not a conflict, return an error 2
 		if errors.As(err, &exitErr) && exitErr.ExitCode() == 1 {
-			log.Warn("Merge completed with conflicts. Manual resolution required.")
 			hasConflicts = true
+			if !opts.Force {
+				log.Error("Merge conflicts detected. Resolve them manually in the merge branch.")
```

Comment:
```suggestion
				log.Warn("Merge completed with conflicts. Manual resolution is required.")
				log.Warn("After resolving the conflicts, run the following command:")
				log.Warn("    make manifests generate fmt vet lint-fix")
				log.Warn("This ensures manifests and generated files are up to date, and the project layout remains consistent.")
```

Reply:
It is not an Error it is a expected behaviour
Also, we must tell about the makefile target those must be regenerated